draft-ietf-scitt-architecture.md

```diff
@@ -364,8 +363,10 @@ Most of the details of the Receipt's contents are specified in the COSE Signed M
     / Collect Receipts /      /   Replay Log   /
    '------------------'      '----------------'
 ~~~
+{: #fig-concept-relationship title="Relationship of Concepts in SCITT"}
 
-This section describes at a high level, the three main roles and associated processes in SCITT: Issuers and Signed Statements, Transparency Service and the Signed Statement Registration process, as well as Relying Parties of the Transparent Statements and the Receipt validation process.
+This section describes at a high level, the three main roles and associated processes in SCITT:
```

Comment:
👍 for this section, lines 366-369.
If we could separate these changes from the architecture changes above, I'd LGTM these lines